Makefile

```diff
@@ -60,6 +60,14 @@ $(hooks):
 hooks: $(hooks)
 
 deploy:
+	@echo "Deploying private/$(BRANCH) to $(RELEASE)..."
+
+	@git fetch private
+
+	@if [ -z $(git rev-list private/$(BRANCH)..oss/$(BRANCH)) ]; then \
+		echo "There are commits in oss/$(BRANCH) that are not in private/$(BRANCH). Please sync the remotes before deploying."; \
+		exit 1; \
+	fi
```

Comment:
one more heads up, the deploy script changed a bit to only deploy from the `private` remote and to first check that  the `oss` and `private` remotes are all set up correctly